mds/db_helpers.py

```diff
@@ -137,7 +137,8 @@ def serialize(event_record):
             # The same encoder as in the model
             "properties": json.dumps(event_record.properties, cls=encoders.JSONEncoder),
             "source": source,
-            "first_saved_at": event_record.first_saved_at,
+            # The model field was renamed, not the table field
+            "first_saved_at": event_record.publication_time,
```

Comment:
I would use `"publication_time"` as dict key here and put `%(publication_time)s,`in the raw sql query

Reply:
I ended up with `"first_saved_at=%(publication_time)s" and I didn't like it.